go/pkg/api/api.go

```diff
@@ -77,12 +77,9 @@ func (b BaseAccount) pubkey() string {
 // Contains info about pagination for large sets of data
 // swagger:model Pagination
 type Pagination struct {
-	// required: true
-	// example: 1
-	Page int `json:"page"`
-	// required: true
-	// example: 10
-	TotalPages int `json:"totalPages"`
+	// required: false
```

Comment:
Wouldn't the cursor be required?

Reply:
> Wouldn't the cursor be required?

No. If `cursor` is not set, then the caller is asking for the first page.

Reply:
This struct is for the return response, but I guess we might be using empty cursor as a way to say no more data instead of a `hasMore` key or something?

Reply:
Not sure what the best practice is for cursor based.

Reply:
yep, no cursor input equals page 1, no cursor output = last page. open to other suggestions here